api/debug/flags.go

```diff
@@ -32,73 +32,94 @@ import (
 	"github.com/klaytn/klaytn/log/term"
 	colorable "github.com/mattn/go-colorable"
 	"gopkg.in/urfave/cli.v1"
+	"gopkg.in/urfave/cli.v1/altsrc"
 )
 
 var Memsize memsizeui.Handler
 
 var (
 	verbosityFlag = cli.IntFlag{
-		Name:  "verbosity",
-		Usage: "Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail",
-		Value: 3,
+		Name:   "verbosity",
+		Usage:  "Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail",
+		Value:  3,
+		EnvVar: "KLAYTN_VERBOSITY",
```

Comment:
All flags have the same prefix `KLAYTN_`, so it seems meaningless

Reply:
Yes, but I think it can be distinguished from environment variables for other applications.

Reply:
This was already discussed in [the above.](https://github.com/klaytn/klaytn/pull/1580#discussion_r941351200)